src/main/java/org/opensearch/flowframework/transport/CreateWorkflowTransportAction.java

```diff
@@ -73,6 +83,21 @@ protected void doExecute(Task task, WorkflowRequest request, ActionListener<Work
             request.getTemplate().workflows(),
             user
         );
+
+        if (request.isDryRun()) {
```

Comment:
We need some test to test the dryRun flag and if validate is working well

Reply:
yup will add tests as I address the PR comments